### PR TITLE
chore: add contribution guidelines

### PR DESCRIPTION
**Pull request by virkt25**
chore: add contribution guidelines

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Contributing to LoopBack
+
+This document lists the differences between the contribution guidelines for
+this repository and the general [**Contributing to LoopBack**](http://loopback.io/doc/en/contrib/index.html) guidelines.
+
+### Developer Certificate of Origin
+
+This repository uses a [Developer Certificate of Origin (DCO)]() instead of a
```
Comment (shimks) [inline CONTRIBUTING.md]: Missing link here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+[Contributor License Agreement]() like most other LoopBack repositories. DCO
```
Comment (shimks) [inline CONTRIBUTING.md]: Missing link here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+is an easier process to adhere to. [Full text of DCO] can be found below, formatted
```
Comment (shimks) [inline CONTRIBUTING.md]: Missing link here

**Review by shimks — APPROVED** [review]
LGTM once the links have been fixed

**Review by b-admike — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (dhmlau) [reply CONTRIBUTING.md]: https://developercertificate.org/

**Comment by dhmlau** [comment]
I wonder if it would be helpful to have instruction for people who has committed but without signing, what they should do next -- put in random changes so that they can commit using `-s` option?  It would be helpful who are not familiar with git, like me.

**Review by shimks — APPROVED** [review]

**Review by dhmlau — APPROVED** [review]
thanks!

**Pull request merged**